packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -74,8 +74,8 @@ export class Breadcrumb extends BaseComponent<IBreadcrumbProps, IBreadcrumbState
 
     return (
       <div className={ css('ms-Breadcrumb', className, styles.root) } ref='renderingArea'>
-        <FocusZone direction={ FocusZoneDirection.horizontal }>
-          <ul className={ css('ms-Breadcrumb-list', styles.list) }>
+        <FocusZone direction={ FocusZoneDirection.horizontal } role='navigation'>
```

Comment:
@cschleiden I can't find any documentation about a limit of one navigation landmarks per page, I don't see why you couldn't have multiple sections of links. I think the question is do we want to try to mimic this new documentation just coming out or classify the breadcrumbs as tabs